avocado/utils/cpu.py

```diff
@@ -114,3 +115,14 @@ def get_cpu_arch():
         return 'x86_64'
     else:
         return 'i386'
+
+
+def cpu_online_list():
+    """
+    Check out the available cpu online list
+    """
+    cpus = []
+    for line in open('/proc/cpuinfo', 'r'):
+        if line.startswith('processor'):
+            cpus.append(line.split()[2])  # grab cpu number
```

Comment:
Yes, list of cpus is nice, I'm wondering whether it'd be better to store it as integers, rather than strings. No strong opinion about it, so please consider it and either add `cpus.append(int(line.split()[2]))` or leave it as is.